### ISSUE TITLE
val_acc is wrong

### ISSUE THREAD

**Issue by DL-Chang** [rajatvikramsingh/cifar10-vgg16#1]
you use sigmoid as the activation function for the last layer and binary_crossentropy as loss。 Therefore, the displayed val_acc is not the val_acc of all test data. I repeated your experiment and calculated the actual val_acc(About 0.62).

**Comment by rajatvikramsingh** [comment]
Hey, thanks for your feedback. Did you use the categorical cross entropy to get that accuracy?

**Comment by DL-Chang** [comment]
No, I use the sigmoid and  binary_crossentropy to train the model, and a 90% accuracy rate has been achieved on val_acc. Then, I use the  model that has been trained to predict the label of the test data, and compare with the true label to get the accuracy(is 62%).
